stock_quant_manual_assign/wizard/assign_manual_quants_view.xml

```diff
@@ -14,7 +14,11 @@
                         <field name="location_id" />
                         <field name="on_hand" force_save="1" />
                         <field name="reserved" force_save="1" />
```

Comment:
```suggestion
                        <field name="on_hand" />
                        <field name="reserved" />
```

I think we should remove `false_save="1"` to avoid future confusions.

Reply:
My bad, I misunderstood the history. No, no need to remove `false_save="1"`.